agent/agent_test.go

```diff
@@ -246,6 +246,63 @@ func TestAgent_ReconnectConfigSettings(t *testing.T) {
 	}()
 }
 
+func TestAgent_HTTPMaxHeaderBytes(t *testing.T) {
+	tests := []struct {
+		name                 string
+		maxHeaderBytes       int
+		expectError          bool
```

Comment:
I guess this `expectError` field was copied from another test case? It does not look to be used here (it's always false), so I think it can be removed.

Reply:
Yeah, I see that now.  Removed